scripts/claude-mem-bridge.sh

```
@@ -24,7 +24,7 @@ claude_mem_search() {
     local limit="${2:-5}"
     local project="${3:-}"
 
-    local url="${CLAUDE_MEM_URL}/api/search?query=$(python3 -c "import urllib.parse; print(urllib.parse.quote('$query'))" 2>/dev/null || echo "$query")&limit=${limit}"
+    local url="${CLAUDE_MEM_URL}/api/search?query=$(python3 -c "import sys,urllib.parse; print(urllib.parse.quote(sys.argv[1]))" "$query" 2>/dev/null || printf '%s' "$query")&limit=${limit}"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)